### DISCUSSION TITLE
Timelord ips question

### DISCUSSION THREAD

**Discussion by isipisi89** [Chia-Network/chia-blockchain#6717] [Q&A]
Hello together, i am running a timelord on a node with an ryzen 5 2600x @4ghz.
The timelord test is getting 135k ips a second. 

Is this fast enough, or should I rather stop the timelord ?

I alsow testet a i9 9900hk and this cpu gets 193k ips a second.
Should i take the i9 or is the ryzen ok ?

**Comment by loppefaaret** [comment]
here is the thing, all the timelords connected to the network are basically running 3 cores on full blast - and the timelord that gets to process the data first, is the one that submits it first - that makes all the other timelords data obsolete - so unless you can consitently fight with a 11900k overclocked, there really isn't much sense in running a regular network timelord!
now, there are some "bluebox" config option of the timelord, where instead of working on the tip, creating VDF, it takes old blocks, and compresses them, and propagates these compressed old blocks to the network, shrinking the block sizes of all the nodes, and makes it easier for any new nodes that needs to sync this data!
you can find a bit more on this here:
https://github.com/Chia-Network/chia-blockchain/wiki/Timelords

just remember to keep an eye on the node the timelord is connected to, it can at some points fall out of sync, if it can't get CPU time when it needs it!

**Reply by isipisi89** [reply]
Ok thx. 11900k gets round about 215-230k ips. I think running an 193k ips timelord local is ok ish.

**Reply by loppefaaret** [reply]
yeah, thats quite respectable

**Reply by isipisi89** [reply]
> yeah, thats quite respectable

How can i check wich timrlord my node is conected with?

**Reply by loppefaaret** [reply]
a timelord is pretty much like a farmer in that regard, it connects to the "full_node_peer:" in `config.yaml` under the "timelord:" section

**Reply by isipisi89** [reply]
so every connected node could be a timrlord too.

**Reply by loppefaaret** [reply]
no, their connection attempt would not have their keys based on the same CA - and the attempt would fail

**Reply by isipisi89** [reply]
So 1 more question, when i am running a timelord on my local network dose my full_node/farmer uses only the timelord wich is connected to it?

Or dose it use multiple timelords on the network?
My timelord is running with 193k ips.

So lets say my harvester found a proof, is this proof only send to my own timelord or alsow to the timelords on the internet?

**Reply by loppefaaret** [reply]
the proof is handed off to the farmer, the farmer turns it over to the node, and the node will immediatly propagate it to all connected full_node peers, as well as you local timelord
the full_node listens to anything from you local timelord, and anything coming in from other full_nodes, it will also receive data from other timelords, and should even pass necessary info to your timelord, such as block ######## is done
